### ISSUE TITLE
Making datepicker screen reader friendly

### ISSUE THREAD

**Issue by mrgodhani** [chemerisuk/better-dateinput-polyfill#129]
Currently datepicker seems to be keyboard friendly accessible. But when it comes to screenreader it's not that accessible friendly. It is missing aria labels for dates. 

Example: https://dequeuniversity.com/library/aria/date-picker

Is there anyway to include correct aria-labels for the dates to also make it screen reader friendly?
